packages/ERTP/src/displayInfo.js

```diff
@@ -48,7 +45,11 @@ export const assertDisplayInfo = allegedDisplayInfo => {
   assertKeysAllowed(displayInfoKeys, allegedDisplayInfo);
 };
 
-export const coerceDisplayInfo = allegedDisplayInfo => {
+/**
+ * @param {any} [allegedDisplayInfo={}]
```

Comment:
Can we set this type to `DisplayInfo`? I'd rather users fail-fast in linting than discover a problem during execution.

Reply:
This was tricky to do, but I finally found a combination that worked around Typescript's suboptimal handling of types of optional arguments that are reassigned.